web3.js/src/stake-program.js

```diff
@@ -388,6 +429,15 @@ export const STAKE_INSTRUCTION_LAYOUTS = Object.freeze({
     index: 5,
     layout: BufferLayout.struct([BufferLayout.u32('instruction')]),
   },
+  AuthorizeWithSeed: {
+    index: 6,
```

Comment:
```suggestion
    index: 8,
```

Reply:
Haven't been keeping up with all of the stake instructions here, you'll need to skip the indices that are reserved for `SetLockup` and `Merge`